Code/Editor/EditorModularViewportCameraComposer.cpp

```diff
@@ -311,4 +345,62 @@ namespace SandboxEditor
                 m_viewportId, &AtomToolsFramework::ModularViewportCameraControllerRequestBus::Events::ClearReferenceFrame);
         }
     }
+
+    void EditorModularViewportCameraComposer::OnTick(const float deltaTime, [[maybe_unused]] AZ::ScriptTimePoint time)
+    {
+        const float delta = deltaTime / ed_cameraDefaultOrbitFadeDuration;
+
+        if (m_defaultOrbiting)
+        {
+            m_defaultOrbitOpacity = AZStd::min(m_defaultOrbitOpacity + delta, 1.0f);
+        }
+        else
+        {
+            m_defaultOrbitOpacity = AZStd::max(m_defaultOrbitOpacity - delta, 0.0f);
+            if (m_defaultOrbitOpacity == 0.0f)
+            {
+                AZ::TickBus::Handler::BusDisconnect();
+                AzFramework::ViewportDebugDisplayEventBus::Handler::BusDisconnect();
+            }
+        }
+
+        m_defaultOrbiting = false;
```

Comment:
The logic of the updating `m_defaultOrbitOpacity` increasing with delta will only be done once, since the boolean is always set to false at the end. Just double checking that's what is expecting.

Reply:
`m_defaultOrbitOpacity` is set to true in the pivot point callback, so is always set to true while 'Alt' is held, this is just to ensure when Alt is released, we fade out the orbit point

Reply:
I see, thanks for the clarification.